features/guidance/exit-numbers-names.feature

```diff
@@ -41,8 +41,8 @@ Feature: Exit Numbers and Names
             | ef     | motorway_link | ExitRamp |              |
 
        When I route I should get
-            | waypoints | route                      | turns                               | exits   |
-            | a,f       | MainRoad,ExitRamp,ExitRamp | depart,off ramp slight right,arrive | ,10;12, |
+            | waypoints | route                      | turns                               | exits    |
+            | a,f       | MainRoad,ExitRamp,ExitRamp | depart,off ramp slight right,arrive | ,10; 12, |
```

Comment:
from the discussions around https://github.com/Project-OSRM/osrm-backend/issues/4270 I presume we will stay with `;`. So I guess this is just what we should do in terms of string handling and consistency.